documentation/modules/rn-2.5.adoc

```diff
@@ -26,27 +26,32 @@ In this version of {project-short}, migration using OpenStack source providers g
 
 .Disabling FIPS
 
-EMS enforcement is disabled for migrations with VMware vSphere source providers to enable migrations from versions of vSphere that are supported by {project-short} but do not comply with the 2023 FIPS requirements.
+With VMware vSphere source providers, Enterprise Master Secret (EMS) enforcement is disabled for migrations to enable migrations from versions of vSphere that are supported by {project-short} but do not meet the 2023 FIPS requirements.
 
 .Integration of the create and update provider user interface
 
 The user interface of create and update providers now aligns with the look and feel of the {ocp} web console and displays up-to-date data.
 
 .Standalone UI
 
-The old UI of MTV 2.3 cannot be enabled by setting `feature_ui: true` in ForkliftController anymore.
+The old UI of {project-short} 2.3 cannot be enabled by setting `feature_ui: true` in ForkliftController anymore.
 
-.Errors logged in populator pods are improved
+.Support deployment on OpenShift 4.15
+
+A vulnerability in Node.js Package Manager (npm) IP package allowed an attacker to run code and obtain sensitive information. This issue has been resolved in {project-short} 2.5.6 by upgrading the package `node-ip` to version 2.0.1
+
+For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=2265161[CVE-2023-42282]
```

Comment:
Why are you linking to Bugzilla when you have had the correct link shared to the Red Hat CVE db.